source/_UIAHandler.py

```diff
@@ -206,6 +211,9 @@ def MTAThreadFunc(self):
 			self.clientObject.AddPropertyChangedEventHandler(self.rootElement,TreeScope_Subtree,self.baseCacheRequest,self,UIAPropertyIdsToNVDAEventNames.keys())
 			for x in UIAEventIdsToNVDAEventNames.iterkeys():  
 				self.clientObject.addAutomationEventHandler(x,self.rootElement,TreeScope_Subtree,self.baseCacheRequest,self)
+			# #7984: add support for notification event (IUIAutomation5, part of Windows 10 build 16299 and later).
+			if IUIAVersion >= "IUIAutomation5":

```

Comment:
Hmm, even though Python uses lexicographical comparison for strings and therefore this is code that does the write thing, I'd feel safer if you'd avoid this. As @michaeldcurran pointed out in https://github.com/nvaccess/nvda/pull/8045#issuecomment-369188217, all UIA interfaces inherit from the one before, so you could do an instance check on the client object for IUIAutomation5.